deploy/helm/scf/assets/operations/instance_groups/doppler.yaml

```diff
@@ -35,16 +35,61 @@
     - name: log-cache-proxy # log-cache-cf-auth-proxy
       protocol: TCP
       internal: 8083
+    run:
+      healthcheck:
+        doppler:
+          readiness:
+            exec:
+              command: [curl, --fail, --head, --silent, http://localhost:14825/health]
+          liveness:
+            exec:
+              command: [pidof, doppler]
 
 # Add quarks properties for log-cache.
+- type: replace
+  path: /instance_groups/name=doppler/jobs/name=log-cache/properties/health_addr?
+  value: "::6060"
```

Comment:
Why use IPv6 notation here?

Reply:
Mainly because I'm lazy and wanted to type less :p I guess now that I have to explain it I've actually had to type more…

Reply:
I was just curious if there was another reason that I was not seeing. :P
Sorry for making you type more. haha